tools/clang/test/SemaHLSL/hlsl/intrinsics/barrier/barrier-cs-sm67-compat.hlsl

```diff
@@ -0,0 +1,103 @@
+// RUN: %dxc -Tlib_6_7 -verify %s
+// REQUIRES: dxil-1-8
```

Comment:
Is this correct? This is a validator test so it doesn't seem like it should require the 1.8 validator.

Reply:
Why do we have any requirement on the validator? The validator doesn't run as part of this test. `-verify` stops at parsing.

Reply:
I think these can be resolved, since I've modified the tests as planned.  Re-activate if you think otherwise.